src/Authorization/AuthorizatorInterface.php

```diff
@@ -13,6 +13,9 @@
 {
     /**
      * @param string|IResource $resource
+     * @param string           $privilege
+     *
+     * @return bool
```

Comment:
These shouldn't be necessary, they just duplicate the typehints. Does removing them cause any error?